pkg/transformer/network_test.go

```diff
@@ -172,16 +169,6 @@ metadata:
   namespace: default
 spec:
   rules:
-  - host: example.com
```

Comment:
Whey remove this rule? It tests not to override matched rule and insert new one.